indico/modules/events/api.py

```diff
@@ -719,7 +719,7 @@ def contribution(self, idlist):
 
         def _iterate_objs(objIds):
             for objId in objIds:
-                obj = event.contributions.filter_by(id=objId).one()
+                obj = Contribution.query.with_parent(event).filter_by(id=objId).one()
```

Comment:
AFAIK the code afterwards still tries to use the old fossils which are not suitable for the new objects, i.e. just fixing the query here won't fix that export